support/k8s/k8s-workload-registrar/config_crd.go

```diff
@@ -71,8 +73,21 @@ func (c *CRDMode) ParseConfig(hclConfig string) error {
 	}
 
 	// Eliminate reference to the non-existing context (strip out the blank space first).
```

Comment:
Does this comment need to be updated?

Reply:
Yeah this could probably be made clearer.